jspm/index.html

```diff
@@ -7,7 +7,7 @@
 <body>
     <div id="content"></div>
     <script>
-        System.import('app').then(function(m) {
+        SystemJS.import("app").then(function(m) {
```

Comment:
Yes it `System` still exists but it will be deprecated in the next major version. Most documentation has been updated to use `SystemJS`. You can read about the change in systemjs/systemjs/issues/936